templates/zerver/help/import-from-slack.md

```diff
@@ -19,14 +19,13 @@ following steps, which are described in more detail below:
 
 {start_tabs}
 
-1. [Export your Slack data](/help/import-from-slack#export-your-slack-data).
+1. [Export your Slack data](#export-your-slack-data).
```

Comment:
@timabbott do you have a preference for how we do anchor links within a page?

Reply:
This format is correct.